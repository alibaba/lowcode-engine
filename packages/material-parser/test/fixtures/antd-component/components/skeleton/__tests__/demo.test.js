import demoTest from '../../../tests/shared/demoTest';

demoTest('skeleton');
