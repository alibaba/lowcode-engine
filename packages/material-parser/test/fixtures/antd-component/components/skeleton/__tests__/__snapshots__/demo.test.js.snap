// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./components/skeleton/demo/active.md correctly 1`] = `
<div
  class="ant-skeleton ant-skeleton-active"
>
  <div
    class="ant-skeleton-content"
  >
    <h3
      class="ant-skeleton-title"
      style="width:38%"
    />
    <ul
      class="ant-skeleton-paragraph"
    >
      <li />
      <li />
      <li
        style="width:61%"
      />
    </ul>
  </div>
</div>
`;

exports[`renders ./components/skeleton/demo/basic.md correctly 1`] = `
<div
  class="ant-skeleton"
>
  <div
    class="ant-skeleton-content"
  >
    <h3
      class="ant-skeleton-title"
      style="width:38%"
    />
    <ul
      class="ant-skeleton-paragraph"
    >
      <li />
      <li />
      <li
        style="width:61%"
      />
    </ul>
  </div>
</div>
`;

exports[`renders ./components/skeleton/demo/children.md correctly 1`] = `
<div
  class="article"
>
  <div>
    <h4>
      Ant Design, a design language
    </h4>
    <p>
      We supply a series of design principles, practical patterns and high quality design resources (Sketch and Axure), to help people create their product prototypes beautifully and efficiently.
    </p>
  </div>
  <button
    class="ant-btn"
    type="button"
  >
    <span>
      Show Skeleton
    </span>
  </button>
</div>
`;

exports[`renders ./components/skeleton/demo/complex.md correctly 1`] = `
<div
  class="ant-skeleton ant-skeleton-with-avatar"
>
  <div
    class="ant-skeleton-header"
  >
    <span
      class="ant-skeleton-avatar ant-skeleton-avatar-lg ant-skeleton-avatar-circle"
    />
  </div>
  <div
    class="ant-skeleton-content"
  >
    <h3
      class="ant-skeleton-title"
      style="width:50%"
    />
    <ul
      class="ant-skeleton-paragraph"
    >
      <li />
      <li />
      <li />
      <li />
    </ul>
  </div>
</div>
`;

exports[`renders ./components/skeleton/demo/element.md correctly 1`] = `
<div>
  <div>
    <form
      class="ant-form ant-form-inline"
      style="margin-bottom:16px"
    >
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-label"
        >
          <label
            class=""
            title="ButtonActive"
          >
            ButtonActive
          </label>
        </div>
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <button
                aria-checked="false"
                class="ant-switch"
                role="switch"
                type="button"
              >
                <span
                  class="ant-switch-inner"
                />
              </button>
            </div>
          </div>
        </div>
      </div>
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-label"
        >
          <label
            class=""
            title="ButtonSize"
          >
            ButtonSize
          </label>
        </div>
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <div
                class="ant-radio-group ant-radio-group-outline"
              >
                <label
                  class="ant-radio-button-wrapper ant-radio-button-wrapper-checked"
                >
                  <span
                    class="ant-radio-button ant-radio-button-checked"
                  >
                    <input
                      checked=""
                      class="ant-radio-button-input"
                      type="radio"
                      value="default"
                    />
                    <span
                      class="ant-radio-button-inner"
                    />
                  </span>
                  <span>
                    Default
                  </span>
                </label>
                <label
                  class="ant-radio-button-wrapper"
                >
                  <span
                    class="ant-radio-button"
                  >
                    <input
                      class="ant-radio-button-input"
                      type="radio"
                      value="large"
                    />
                    <span
                      class="ant-radio-button-inner"
                    />
                  </span>
                  <span>
                    Large
                  </span>
                </label>
                <label
                  class="ant-radio-button-wrapper"
                >
                  <span
                    class="ant-radio-button"
                  >
                    <input
                      class="ant-radio-button-input"
                      type="radio"
                      value="small"
                    />
                    <span
                      class="ant-radio-button-inner"
                    />
                  </span>
                  <span>
                    Small
                  </span>
                </label>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-label"
        >
          <label
            class=""
            title="ButtonShape"
          >
            ButtonShape
          </label>
        </div>
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <div
                class="ant-radio-group ant-radio-group-outline"
              >
                <label
                  class="ant-radio-button-wrapper ant-radio-button-wrapper-checked"
                >
                  <span
                    class="ant-radio-button ant-radio-button-checked"
                  >
                    <input
                      checked=""
                      class="ant-radio-button-input"
                      type="radio"
                      value="default"
                    />
                    <span
                      class="ant-radio-button-inner"
                    />
                  </span>
                  <span>
                    Default
                  </span>
                </label>
                <label
                  class="ant-radio-button-wrapper"
                >
                  <span
                    class="ant-radio-button"
                  >
                    <input
                      class="ant-radio-button-input"
                      type="radio"
                      value="round"
                    />
                    <span
                      class="ant-radio-button-inner"
                    />
                  </span>
                  <span>
                    Round
                  </span>
                </label>
                <label
                  class="ant-radio-button-wrapper"
                >
                  <span
                    class="ant-radio-button"
                  >
                    <input
                      class="ant-radio-button-input"
                      type="radio"
                      value="circle"
                    />
                    <span
                      class="ant-radio-button-inner"
                    />
                  </span>
                  <span>
                    Circle
                  </span>
                </label>
              </div>
            </div>
          </div>
        </div>
      </div>
    </form>
    <div
      class="ant-skeleton ant-skeleton-element"
    >
      <span
        class="ant-skeleton-button"
      />
    </div>
  </div>
  <br />
  <div>
    <form
      class="ant-form ant-form-inline"
      style="margin-bottom:16px"
    >
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-label"
        >
          <label
            class=""
            title="AvatarActive"
          >
            AvatarActive
          </label>
        </div>
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <button
                aria-checked="false"
                class="ant-switch"
                role="switch"
                type="button"
              >
                <span
                  class="ant-switch-inner"
                />
              </button>
            </div>
          </div>
        </div>
      </div>
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-label"
        >
          <label
            class=""
            title="AvatarSize"
          >
            AvatarSize
          </label>
        </div>
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <div
                class="ant-radio-group ant-radio-group-outline"
              >
                <label
                  class="ant-radio-button-wrapper ant-radio-button-wrapper-checked"
                >
                  <span
                    class="ant-radio-button ant-radio-button-checked"
                  >
                    <input
                      checked=""
                      class="ant-radio-button-input"
                      type="radio"
                      value="default"
                    />
                    <span
                      class="ant-radio-button-inner"
                    />
                  </span>
                  <span>
                    Default
                  </span>
                </label>
                <label
                  class="ant-radio-button-wrapper"
                >
                  <span
                    class="ant-radio-button"
                  >
                    <input
                      class="ant-radio-button-input"
                      type="radio"
                      value="large"
                    />
                    <span
                      class="ant-radio-button-inner"
                    />
                  </span>
                  <span>
                    Large
                  </span>
                </label>
                <label
                  class="ant-radio-button-wrapper"
                >
                  <span
                    class="ant-radio-button"
                  >
                    <input
                      class="ant-radio-button-input"
                      type="radio"
                      value="small"
                    />
                    <span
                      class="ant-radio-button-inner"
                    />
                  </span>
                  <span>
                    Small
                  </span>
                </label>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-label"
        >
          <label
            class=""
            title="AvatarShape"
          >
            AvatarShape
          </label>
        </div>
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <div
                class="ant-radio-group ant-radio-group-outline"
              >
                <label
                  class="ant-radio-button-wrapper"
                >
                  <span
                    class="ant-radio-button"
                  >
                    <input
                      class="ant-radio-button-input"
                      type="radio"
                      value="square"
                    />
                    <span
                      class="ant-radio-button-inner"
                    />
                  </span>
                  <span>
                    Square
                  </span>
                </label>
                <label
                  class="ant-radio-button-wrapper ant-radio-button-wrapper-checked"
                >
                  <span
                    class="ant-radio-button ant-radio-button-checked"
                  >
                    <input
                      checked=""
                      class="ant-radio-button-input"
                      type="radio"
                      value="circle"
                    />
                    <span
                      class="ant-radio-button-inner"
                    />
                  </span>
                  <span>
                    Circle
                  </span>
                </label>
              </div>
            </div>
          </div>
        </div>
      </div>
    </form>
    <div
      class="ant-skeleton ant-skeleton-element"
    >
      <span
        class="ant-skeleton-avatar ant-skeleton-avatar-circle"
      />
    </div>
  </div>
  <br />
  <div>
    <form
      class="ant-form ant-form-inline"
      style="margin-bottom:16px"
    >
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-label"
        >
          <label
            class=""
            title="InputActive"
          >
            InputActive
          </label>
        </div>
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <button
                aria-checked="false"
                class="ant-switch"
                role="switch"
                type="button"
              >
                <span
                  class="ant-switch-inner"
                />
              </button>
            </div>
          </div>
        </div>
      </div>
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-label"
        >
          <label
            class=""
            title="InputSize"
          >
            InputSize
          </label>
        </div>
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <div
                class="ant-radio-group ant-radio-group-outline"
              >
                <label
                  class="ant-radio-button-wrapper ant-radio-button-wrapper-checked"
                >
                  <span
                    class="ant-radio-button ant-radio-button-checked"
                  >
                    <input
                      checked=""
                      class="ant-radio-button-input"
                      type="radio"
                      value="default"
                    />
                    <span
                      class="ant-radio-button-inner"
                    />
                  </span>
                  <span>
                    Default
                  </span>
                </label>
                <label
                  class="ant-radio-button-wrapper"
                >
                  <span
                    class="ant-radio-button"
                  >
                    <input
                      class="ant-radio-button-input"
                      type="radio"
                      value="large"
                    />
                    <span
                      class="ant-radio-button-inner"
                    />
                  </span>
                  <span>
                    Large
                  </span>
                </label>
                <label
                  class="ant-radio-button-wrapper"
                >
                  <span
                    class="ant-radio-button"
                  >
                    <input
                      class="ant-radio-button-input"
                      type="radio"
                      value="small"
                    />
                    <span
                      class="ant-radio-button-inner"
                    />
                  </span>
                  <span>
                    Small
                  </span>
                </label>
              </div>
            </div>
          </div>
        </div>
      </div>
    </form>
    <div
      class="ant-skeleton ant-skeleton-element"
    >
      <span
        class="ant-skeleton-input"
        style="width:300px"
      />
    </div>
  </div>
</div>
`;

exports[`renders ./components/skeleton/demo/list.md correctly 1`] = `
<div>
  <button
    aria-checked="false"
    class="ant-switch"
    role="switch"
    type="button"
  >
    <span
      class="ant-switch-inner"
    />
  </button>
  <div
    class="ant-list ant-list-vertical ant-list-lg ant-list-split"
  >
    <div
      class="ant-spin-nested-loading"
    >
      <div
        class="ant-spin-container"
      >
        <ul
          class="ant-list-items"
        >
          <li
            class="ant-list-item ant-list-item-no-flex"
          >
            <div
              class="ant-skeleton ant-skeleton-with-avatar ant-skeleton-active"
            >
              <div
                class="ant-skeleton-header"
              >
                <span
                  class="ant-skeleton-avatar ant-skeleton-avatar-lg ant-skeleton-avatar-circle"
                />
              </div>
              <div
                class="ant-skeleton-content"
              >
                <h3
                  class="ant-skeleton-title"
                  style="width:50%"
                />
                <ul
                  class="ant-skeleton-paragraph"
                >
                  <li />
                  <li />
                </ul>
              </div>
            </div>
          </li>
          <li
            class="ant-list-item ant-list-item-no-flex"
          >
            <div
              class="ant-skeleton ant-skeleton-with-avatar ant-skeleton-active"
            >
              <div
                class="ant-skeleton-header"
              >
                <span
                  class="ant-skeleton-avatar ant-skeleton-avatar-lg ant-skeleton-avatar-circle"
                />
              </div>
              <div
                class="ant-skeleton-content"
              >
                <h3
                  class="ant-skeleton-title"
                  style="width:50%"
                />
                <ul
                  class="ant-skeleton-paragraph"
                >
                  <li />
                  <li />
                </ul>
              </div>
            </div>
          </li>
          <li
            class="ant-list-item ant-list-item-no-flex"
          >
            <div
              class="ant-skeleton ant-skeleton-with-avatar ant-skeleton-active"
            >
              <div
                class="ant-skeleton-header"
              >
                <span
                  class="ant-skeleton-avatar ant-skeleton-avatar-lg ant-skeleton-avatar-circle"
                />
              </div>
              <div
                class="ant-skeleton-content"
              >
                <h3
                  class="ant-skeleton-title"
                  style="width:50%"
                />
                <ul
                  class="ant-skeleton-paragraph"
                >
                  <li />
                  <li />
                </ul>
              </div>
            </div>
          </li>
        </ul>
      </div>
    </div>
  </div>
</div>
`;
